### PR TITLE
azure and k8s mss added

### PR DESCRIPTION
**Pull request by n1654**
Azure-k8s Demo related MSs added http://confluence.ubiqube.com/display/UID/NEC+Americas+K8+cluste

Linux/Generic/K8S_CLI - to manage k8s cluster via ssh
Kubernetes/Generic - library extended

This library supposed to be shared with customer

### CODE REVIEW

File: LINUX/K8S_CLI/k8s_cli_control.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    <configType>cli</configType>
+    <createTemplateId/>
+    <createTemplateObject/>
+    <defaultDisplay>false</defaultDisplay>
+    <dynamic>false</dynamic>
+    <importIfMandatoryPresent>false</importIfMandatoryPresent>
+    <maxInstances>0</maxInstances>
+    <relatedObjects/>
+    <name>k8s_cli_control</name>
+    <group>Default</group>
```
Comment (abr-ubiqube) [inline LINUX/K8S_CLI/k8s_cli_control.xml:18]: can you update the group with something like K8s|CLI

**Review by abr-ubiqube — CHANGES_REQUESTED** [review]
@n1654 
for the LINUX/K8 CLI, maybe we could create a dedicated adapters based on linux?
we did something similar for netboxbased on REST Generic: https://github.com/openmsa/Adapters/tree/master/adapters/rest_netbox

the advantage is that the UI will show something like Linux / K8s CLI and the microservice will be easier to select

**Review by n1654 — COMMENTED** [review]
K8s|CLI added as group name

**Review by abr-ubiqube — APPROVED** [review]

File: LINUX/K8S_CLI/k8s_cli_control.xml (hunk 1 above)
Comment (n1654) [reply LINUX/K8S_CLI/k8s_cli_control.xml]: Updated.

Regarding to DA, I'm not sure if Kubernetes CLI has a stable set of commands - so MS suits better I think. Commands vary from version to version. 
Or you mean just separate Linux/Generic library as a copy in order not to mix them ?

**Pull request merged**